src/index.html

```diff
@@ -1,15 +1,201 @@
+          <input
+           required
+           autocomplete="off"
+           type="text"
+           id="surname"
+           name="Surname"
+          >
+        </div>
+        <div class="field">
+          <label class="label" for="name">Name:</label>
+          <input
+           required
+           autocomplete="off"
+           type="text"
+           id="name"
+           name="name"
+          >
+        </div>
+        <div class="field">
+          <label class="label" for="old">How old are You?</label>
+          <input
+           required
+           autocomplete="off"
+           type="number"
+           id="old"
+           name="old"
+           value="12"
+           min="6"
+           max="100"
+          >
+        </div>
+        <div class="field">
+          <label class="label" for="age">Full date of birth:</label>
+          <input
+           required
+           autocomplete="off"
+           type="date"
+           id="age"
+           name="age"
+          >
+        </div>
+        <div class="field">
+          <label class="label" for="box">I accept the term of the agreement</label>
+          <input
+           required
+           autocomplete="off"
+           type="checkbox"
+           id="box"
+           name="box"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="field">
+          <label class="label" for="email">E-mail:</label>
+          <input
+           required
+           autocomplete="off"
+           type="email"
+           id="email"
+           name="E-mail"
+           placeholder="email@example.com"
+          >
+        </div>
+        <div class="field">
+          <label class="label" for="password">Password:</label>
+          <input
+           required
+           autocomplete="off"
+           type="password"
+           id="password"
+           name="Password"
+           minlength="3"
+           maxlength="15"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="field">
+          <div class="label">
+            Do you love cats?
+            <input
+             type="radio"
+             name="radio"
+             value="Yes"
+             id="Yes"
+            >
+            <label for="Yes">
+              Yes
+            </label>
+            <input
+            type="radio"
+            name="radio"
+            value="No"
+            id="No"
+           >
+            <label for="No">
+            No
+            </label>
+          </div>
+        </div>
+        <div class="field">
+          <label class="label" for="color">What is your favorite color?</label>
+          <input
+           required
+           autocomplete="off"
+           type="color"
+           id="color"
+           name="Color"
+          >
```

Comment:
fix indentation everywhere, it should be 2 spaces, not one
```suggestion
          <input
            required
            autocomplete="off"
            type="color"
            id="color"
            name="Color"
          >
```